py2-matplotlib.spec

```diff
@@ -1,29 +1,42 @@
-### RPM external py2-matplotlib 1.4.3
+### RPM external py2-matplotlib 2.2.4
 ## INITENV +PATH PYTHONPATH %i/${PYTHON_LIB_SITE_PACKAGES}
-Source: http://sourceforge.net/projects/matplotlib/files/matplotlib/matplotlib-%{realversion}/matplotlib-%{realversion}.tar.gz
-Requires: py2-pytz py2-numpy py2-python-dateutil zlib libpng freetype py2-pyparsing py2-six
-BuildRequires: py2-setuptools
+
+Source: https://github.com/matplotlib/matplotlib/archive/v%{realversion}.tar.gz
+Requires: py2-pytz py2-numpy py2-python-dateutil zlib libpng freetype py2-pyparsing py2-six py2-cycler py2-subprocess32 py2-kiwisolver
+#Requires: py2-backports
+BuildRequires: py2-pip
 
 %prep
 %setup -n matplotlib-%{realversion}
 
 cat >> setup.cfg <<- EOF
 [directories]
-basedirlist = ${FREETYPE_ROOT},${LIBPNG_ROOT},${ZLIB_ROOT},${PY2_NUMPY_ROOT},${PY2_PYTZ_ROOT},${PY2_SIX}
+basedirlist = ${FREETYPE_ROOT}:${LIBPNG_ROOT}:${ZLIB_ROOT}:${PY2_NUMPY_ROOT}:${PY2_PYTZ_ROOT}:${PY2_SIX}
 
 [gui_support]
 gtk = False
 gtkagg = False
-tkagg = False
+tkagg = True
 wxagg = False
 macosx = False
 EOF
 
+mkdir no-pkg-config
+(echo '#!/bin/sh'; echo 'exit 1') > no-pkg-config/pkg-config
+chmod +x no-pkg-config/pkg-config
+
 %build
-python setup.py build
+#python setup.py build
```

Comment:
@smuzaffar is this commented out on purpose? I just x-checked against IB/11_0 and it's the same there.

Reply:
@amaltaro this line was needed when we were building it without pip. Now the package is build with pip , so pip internally take care of it.

Reply:
thanks for the confirmation, Shahzad.